flocx_market/tests/unit/db/sqlalchemy/test_api.py

```diff
@@ -89,14 +89,31 @@
                                       project_id='7788')
 
 
-def test_offer_get_all(app, db, session):
+def test_offer_get_found(app, db, session):
+    m = api.offer_create(test_offer_data, scoped_context)
+    assert(api.offer_get(m.marketplace_offer_id, scoped_context))
+
+
+def test_offer_get_not_found(app, db, session):
+    with pytest.raises(Exception) as excinfo:
+        api.offer_get("NotHere", scoped_context)
+    assert(excinfo.value.code == 500)
+
+
+def test_offer_get_all_found(app, db, session):
     api.offer_create(test_offer_data, scoped_context)
     api.offer_create(test_offer_data_2, scoped_context)
     api.offer_create(test_offer_data_3, scoped_context_2)
 
     assert len(api.offer_get_all(scoped_context)) == 3
 
 
+def test_offer_get_all_none_found(app, db, session):
+    with pytest.raises(Exception) as excinfo:
```

Comment:
You should be testing for a *specific* exception here, not a generic `Exception`.  Ditto for all the other tests using `pytest.raises`.